_specs/ecip-1089.md

```diff
@@ -0,0 +1,81 @@
+---
+lang: en
+ecip: 1089
+title: Phoenix EVM and Protocol Upgrades (Classic Option)
+status: Draft
+type: Meta
+author: /raw PONG _GHMoaCXLT (@q9f)
+created: 2020-02-24
+discussions-to: https://github.com/ethereumclassic/ECIPs/issues/262
+---
+
+### Simple Summary
+
+Enable the outstanding Ethereum Foundation _Istanbul_ network protocol upgrades on the Ethereum Classic network in a hard-fork code-named _Phoenix_ to enable maximum compatibility across these networks.
+
+This is the _classic_ option equaling to an _Istanbul_ option without gas repricing.
+
+### Abstract
+
+Add support for a subset of protocol-impacting changes introduced in the Ethereum Foundation (ETH) network via the _Istanbul_ hardforks. The proposed changes for Ethereum Classic's _Phoenix_ upgrade include:
+
+- Add Blake2 compression function `F` precompile
+- Reduce alt_bn128 precompile gas costs
+- Net gas metering for SSTORE without dirty maps
+- Add ChainID opcode
+- Disable SSTORE with gasleft lower than call stipend
+- Calldata gas cost reduction
+- Add SELFBALANCE opcode
```

Comment:
Can we have a clear description of what is to NOT be included?

Reply:
- Do not add modify opcode `TIMESTAMP` to opcode `KITTEN` at `0x42`, despite popular opinion.

Reply:
But actually, contextualized comparisons of features should be in the discussions, not the spec.

Reply:
This just sums up the included specifications. I don't know how to improve this section but happy for any suggestions.

Reply:
My opinion is that if we reference the EIP, and modify it, we should bullet point whats changing no?

Reply:
oh hmm i see it below nvm